tests/installation/installer_desktopselection.pm

```diff
@@ -52,6 +54,14 @@ sub run {
     }
     send_key $cmd{next};
 
+    # On leap 42.3 we don't have addon products page, and provide urls as addon
+    # as boot parameter. Trusting gpg key is the done after we click next
+    # on Desktop selection screen
+    if (addon_products_is_applicable() && leap_version_at_least('42.3')) {
+        assert_screen 'import-untrusted-gpg-key-598D0E63B3FD7E48';
+        send_key "alt-t", 1;    # confirm import (trust) key
```

Comment:
please don't use the wait_idle argument on send_key. I think you don't need it at all because there is either an assert_screen coming afterwards or the module ends and the next one would start with an assert_screen